datasets/loopfietsnetwerk/dataset.json

```diff
@@ -0,0 +1,283 @@
+{
+    "type": "dataset",
+    "id": "loopfietsnetwerk",
+    "title": "loopfietsnetwerk",
+    "description": "Topologisch consistent netwerk bedoeld voor het uitvoeren van ruimtelijke netwerkanalyses in het gemeentelijke domein",
+    "spatialDescription": "Gemeente Amsterdam, inclusief Weesp",
+    "license": "public",
+    "status": "beschikbaar",
+    "version": "0.0.1",
+    "theme": [
+      "Ruimte en Topografie",
+      "Verkeer",
+      "Wonen"
+    ],
+    "owner": "Gemeente Amsterdam",
+    "auth": "OPENBAAR",
+    "authorizationGrantor": "Onderzoek & Statistiek",
+    "creator": "Onderzoek & Statistiek",
+    "publisher": "Datadiensten Gemeente Amsterdam",
```

Comment:
Onder `publisher` kan het betreffende datateam ingevuld worden. In dit geval `Datateam Basisstatistiek`, toch?

Reply:
Goed plan. Wordt aangepast

Reply:
Aanpassingen doorgevoerd..